5.5/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
```

Comment:
That was my idea actually... I found having this in Dockerfile pretty ugly and hard to read. Since he is adding the 'mysql' directory in 2 steps before, the create_user.sh addition is noop here.

Also in future, we can put more logic into user creation, etc. Maybe we should make the script accepting parameter (username?).

The script inside the image is nice in my opinion because you can easily inspect how the user was created and with what parameters.